Network/centurylink-cloud-public-ip-listing.md

```diff
@@ -0,0 +1,140 @@
+* In its current iteration setting a source IP filter will secure all public ports, single ports or port ranges specified by the customer. Customers can leverage OS based [firewall services](https://www.ctl.io/cloud-firewall/) if they wish to secure public services in a more granular fashion.
+* Customers are encouraged to leverage the source IP filter unless delivering completely open public Internet services to their user community.
+* Customers should avoid opening RDP or SSH to their virtual machines to the public Internet. As such the following are recommended access methods.
+
+1. Use the free OpenVPN client included in every CenturyLink Cloud Account. Refer to [How To Configure Client VPN](https://www.ctl.io/knowledge-base/network/how-to-configure-client-vpn/). This is the ideal solution for individuals who are mobile and not in fixed office or data center locations.
+2. Build an IPSEC VPN Tunnel from a remote office or data center location. Refer to [Creating a Self-Service IPSEC Site-to-Site VPN Tunnel](https://www.ctl.io/knowledge-base/network/creating-a-self-service-ipsec-site-to-site-vpn-tunnel/). IPSEC VPN tunnels are best for remote access to Cloud Virtual Machines when administrators are in centralized offices or data centers.
+3. If either of the previous options are not feasible customers should at a minimum use the source IP filter service on the public IP and pair that with local OS firewall policies within the guest VM.
+4. Refer to [How to Add Public IP to servers](https://www.ctl.io/knowledge-base/network/how-to-add-public-ip-to-virtual-machine/), for further instructions.
+
+### Public IP Listing
+
+Below is the current CenturyLink public IP listing. It is updated regularly as public IPs are created.
+
+**AU1:**
+    routes:
+        - 65.127.194.144/29
+        - 65.151.184.0/23
+**CA1:**
+    routes:
+        - 216.187.73.144/28
+        - 64.69.71.128/25
+        - 65.39.180.0/24
+        - 65.39.184.0/25
+        - 216.187.110.0/24
+**CA2:**
+    routes:
+        - 70.33.239.96/28
+        - 66.155.96.0/24
+        - 66.155.100.0/24
+        - 69.28.224.128/25
+        - 70.33.208.0/25
+        - 70.33.239.128/25
+        - 107.6.43.0/24
+**CA3:**
+    routes:
+        - 206.152.25.96/28
+        - 206.152.32.0/21
+**DE1:**
+    routes:
+        - 66.155.5.0/28
+        - 66.155.4.0/24
+        - 66.155.94.0/24
+**GB1:**
+    routes:
+        - 176.74.176.144/28
+        - 66.155.18.0/24
+        - 66.155.19.0/24
+        - 66.155.27.0/24
+        - 66.155.28.0/24
+        - 176.74.168.0/25
+        - 176.74.179.0/25
+**GB3:**
+    routes:
+        - 206.142.225.192/28
+        - 206.142.240.0/21
+**IL1:**
+    routes:
+        - 64.94.35.32/28
+        - 64.74.98.0/24
+        - 66.150.98.0/23
+        - 66.150.105.0/24
+        - 66.151.15.0/24
+        - 69.25.149.0/24
+        - 72.5.203.0/24
+        - 74.217.15.0/24
+**NE1:**
```

Comment:
NE1 is a private cloud customer and should not be in a public KB, listing their endpoints